online-shop/src/app/product-list/product-list.component.scss

```diff
@@ -0,0 +1,35 @@
+table {
+  margin-right: auto;
+  margin-left: auto;
+  font-size: 1.25rem;
+  border: 1px solid black;
+  border-collapse: collapse;
+  text-align: center;
+  background-color: antiquewhite;
+}
+
+th {
+  border: 1px solid black;
+  width: 300px;
```

Comment:
In SCSS you can define variables and use those.
For example create a variables.scss file at the top lvl. use it for Theme-colors, default margins, paddings and other things that are reusable